cypress/integration/login.js

```diff
@@ -12,16 +12,19 @@ describe('Sourcebooks login', function() {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Ugne Lekstutiene"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
         cy.get('[aria-label="User"]').click();
         cy.get('[type="submit"]').click();
-
-        cy.url().should('include', '/time-logging');
+        cy.url().should('include', '/time-logging');        
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
+        cy.get('.calendar--today').contains(new Date().getDate());

```

Comment:
This dynamic solution is great! However, this should probably be extracted into a separate test case or if not, the name of the test case should reflect this day assertion, e.g. "... and check if today's date is selected"